packages/expo-cli/src/commands/customize.js

```diff
@@ -0,0 +1,115 @@
+import spawnAsync from '@expo/spawn-async';
+import { ProjectUtils, Web } from '@expo/xdl';
+import chalk from 'chalk';
+import { MultiSelect } from 'enquirer';
+import fs from 'fs';
+import fse from 'fs-extra';
```

Comment:
```suggestion
import fs from 'fs-extra';
```
`fs-extra` includes everything from `fs` too, so you can just import one.